docs/wiki/Deploying-ALZ-HubAndSpoke.md

```diff
@@ -74,12 +74,39 @@ Depending on your requirements, you may choose to deploy additional network infr
 
  ![img](./media/clip_image036b.png)
 
+### Deploying networking resources in an additional region
```

Comment:
should we use the language deploying to multiple regions? I appreciate for portal this is restricted to 2 regions so perhaps for this the language works well, @jtracey93 thoughts?

Reply:
i say leave as is, as it matches the RI option